okta/OKTA (Demo).json

```diff
@@ -0,0 +1 @@
ev-701166.oktapreview.com/api/v1/users/00u9or7keoAfaP4N00h7/lifecycle/reset_password\"},\"self\":{\"href\":\"https://dev-701166.oktapreview.com/api/v1/users/00u9or7keoAfaP4N00h7\"},\"deactivate\":{\"method\":\"POST\",\"href\":\"https://dev-701166.oktapreview.com/api/v1/users/00u9or7keoAfaP4N00h7/lifecycle/deactivate\"}},\"created\":\"2017-02-24T02:19:12.000Z\",\"profile\":{\"firstName\":\"Isaac\",\"lastName\":\"Brock\",\"mobilePhone\":\"555-415-1337\",\"login\":\"isaac.brock@example.com\",\"email\":\"isaac.brock@example.com\"},\"statusChanged\":\"2017-02-24T02:19:12.000Z\",\"id\":\"00u9or7keoAfaP4N00h7\",\"status\":\"PROVISIONED\",\"activated\":\"2017-02-24T02:19:12.000Z\"}","contentType":"application/json"},"model":{"transform":false,"name":"users","swagger":{"suspend":{"id":"suspend","properties":{"method":{"type":"string"},"href":{"type":"string"}}},"resetPassword":{"id":"resetPassword","properties":{"method":{"type":"string"},"href":{"type":"string"}}},"_links":{"id":"_links","properties":{"suspend":{"type":"suspend"},"resetPassword":{"type":"resetPassword"},"self":{"type":"self"},"deactivate":{"type":"deactivate"}}},"credentials":{"id":"credentials","properties":{"provider":{"type":"provider"}}},"provider":{"id":"provider","properties":{"name":{"type":"string"},"type":{"type":"string"}}},"profile":{"id":"profile","properties":{"firstName":{"type":"string"},"lastName":{"type":"string"},"mobilePhone":{"type":"string"},"secondEmail":{"type":"unknown"},"login":{"type":"string"},"email":{"type":"string"}}},"self":{"id":"self","properties":{"href":{"type":"string"}}},"users":{"id":"users","properties":{"lastLogin":{"type":"unknown"},"lastUpdated":{"type":"string"},"passwordChanged":{"type":"unknown"},"_links":{"type":"_links"},"credentials":{"type":"credentials"},"created":{"type":"string"},"profile":{"type":"profile"},"statusChanged":{"type":"string"},"id":{"type":"string"},"activated":{"type":"string"},"status":{"type":"string"}}},"deactivate":{"id":"deactivate","properties":{"method":{"type":"string"},"href":{"type":"string"}}}}},"hooks":[],"modelMigrated":false,"parameters":[{"vendorType":"path","sampleData":"00u9or7keoAfaP4N00h7","dataType":"string","name":"id","description":"The user ID.","vendorDataType":"string","source":"request","type":"path","vendorName":"id","required":true}]}],"protocolType":"http","transformationsEnabled":true,"deleted":false,"hub":"general","typeOauth":false,"name":"OKTA (Demo)","extendable":true,"parameters":[{"vendorType":"header","name":"application/json","source":"request","type":"value","vendorName":"Accept","required":false},{"vendorType":"header","name":"SSWS ${configuration.token}","source":"request","type":"value","vendorName":"Authorization","required":false},{"vendorType":"path","name":"baseurl","source":"request","type":"configuration","vendorName":"baseurl","required":false},{"vendorType":"header","name":"application/json","source":"request","type":"value","vendorName":"Content-Type","required":false}]}
```

Comment:
This also seems to include sensitive info:
```
            "response": {
                "contentType": "application/json",
                "sampleData": "{\"lastUpdated\":\"2017-02-24T02:19:12.000Z\",\"credentials\":{\"provider\":{\"name\":\"OKTA\",\"type\":\"OKTA\"}},\"_links\":{\"suspend\":{\"method\":\"POST\",\"href\":\"https://dev-701166.oktapreview.com/api/v1/users/00u9or7keoAfaP4N00h7/lifecycle/suspend\"},\"resetPassword\":{\"method\":\"POST\",\"href\":\"https://dev-701166.oktapreview.com/api/v1/users/00u9or7keoAfaP4N00h7/lifecycle/reset_password\"},\"self\":{\"href\":\"https://dev-701166.oktapreview.com/api/v1/users/00u9or7keoAfaP4N00h7\"},\"deactivate\":{\"method\":\"POST\",\"href\":\"https://dev-701166.oktapreview.com/api/v1/users/00u9or7keoAfaP4N00h7/lifecycle/deactivate\"}},\"created\":\"2017-02-24T02:19:12.000Z\",\"profile\":{\"firstName\":\"Isaac\",\"lastName\":\"Brock\",\"mobilePhone\":\"555-415-1337\",\"login\":\"isaac.brock@example.com\",\"email\":\"isaac.brock@example.com\"},\"statusChanged\":\"2017-02-24T02:19:12.000Z\",\"id\":\"00u9or7keoAfaP4N00h7\",\"status\":\"PROVISIONED\",\"activated\":\"2017-02-24T02:19:12.000Z\"}"
            },
```

Reply:
done